internal/commands/pull/command_test.go

```diff
@@ -317,6 +334,204 @@ Successfully pulled app down: app
 		assert.Nil(t, err)
 		assert.Equal(t, "<html><body>hello world!</body></html>", string(modifiedData))
 	})
+
+	t.Run("with a template ID to export with", func(t *testing.T) {
+
```

Comment:
[style nit] no need for the newline, could also just inline the `// needs to have...` alongside the `zipPkg, err := ...` line